src/content/docs/en/guides/deploy/clever-cloud.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Deploy your Astro Site to Clever Cloud
+description: How to deploy your Astro site to the web on Clever Cloud.
+type: deploy
+i18nReady: true
+---
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
+
+[Clever Cloud](https://clever-cloud.com) is a European cloud platform that provides automated, scalable services. You can deploy your Astro app with Clever Cloud as a static or Node application.
+
+## Project Configuration
+
+Your Astro project can be deployed on Clever Cloud in two different ways: as a **Static** application, or as a **Node.js** application, regardless of your `output` mode ([pre-rendered or on-demand](https://docs.astro.build/en/basics/rendering-modes/)). However, you'll need less configuration when deploying a Node.js application, since it works out-of-the-box with your `package.json`.
```

Comment:
For internal links we don't use the domain name:
```suggestion
Your Astro project can be deployed on Clever Cloud in two different ways: as a **Static** application, or as a **Node.js** application, regardless of your `output` mode ([pre-rendered or on-demand](/en/basics/rendering-modes/)). However, you'll need less configuration when deploying a Node.js application, since it works out-of-the-box with your `package.json`.
```